helloworld/src/main/js/foo.js

```diff
@@ -1,4 +1,5 @@
 function process() {
+	var foo = 10;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "foo" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar:9000/coding_rules#rule_key=javascript%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of the unused 'foo' variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar:9000/coding_rules#rule_key=javascript%3AUnusedVariable)